### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
```
Comment (vlasiuksasha) [inline src/makeRobotAccountant.js:12]: Function getSum does not take two arguments as per the task description. It should take two arguments, not one.

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -7,6 +7,19 @@
...
+    return function inner(b) {
```
Comment (vlasiuksasha) [inline src/makeRobotAccountant.js:13]: There is no need to create an inner function. getSum should directly take two arguments and return the result or error based on the condition.

File: src/makeRobotAccountant.js [hunk 3]
```diff
@@ -7,6 +7,19 @@
...
+      counter++;
+
+      if (counter % 2 === 0 && counter > 3) {
```
Comment (vlasiuksasha) [inline src/makeRobotAccountant.js:16]: The calculation for determining whether the call is even is incorrect. The count should be checked for divisibility by 4, not 2, to alternate between correct sum and error after the third call.

File: src/makeRobotAccountant.js (hunk 1 above)
Comment (vlasiuksasha) [inline src/makeRobotAccountant.js]: The getSum function should take two arguments according to the task description but it only takes one in your implementation. Please adjust this.

File: src/makeRobotAccountant.js (hunk 2 above)
Comment (vlasiuksasha) [inline src/makeRobotAccountant.js]: The inner function is unnecessary and makes the code more complex. You can simply add the second parameter to the getSum function.

**Review by vlasiuksasha — CHANGES_REQUESTED** [review]
check the comments

**Comment by ghost** [comment]
fixed

File: src/makeRobotAccountant.js (hunk 2 above)
Comment (ghost) [reply src/makeRobotAccountant.js]: fixed

File: src/makeRobotAccountant.js (hunk 1 above)
Comment (ghost) [reply src/makeRobotAccountant.js]: fixed

**Comment by ghost** [comment]
hello @vlasiuksasha please could you have a look at my second commit?

**Pull request closed without merging**